proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -296,89 +300,90 @@ service Microgrid {
 
-  // Return components that have the specified categories only.
-  repeated frequenz.api.common.v1.microgrid.components.ComponentCategory
-    categories = 2;
+  // Return electrical components that have the specified categories only.
+  repeated frequenz.api.common.v1alpha7.microgrid.electrical_components.ElectricalComponentCategory
+    electrical_component_categories = 2;
 }
 
-// A message containing a list of components.
-// Used as the return type in the RPC `ListComponents`.
-message ListComponentsResponse {
-  repeated frequenz.api.common.v1.microgrid.components.Component components = 1;
+// A message containing a list of electrical components.
+// Used as the return type in the RPC `ListElectricalComponents`.
+message ListElectricalComponentsResponse {
+  repeated frequenz.api.common.v1alpha7.microgrid.electrical_components.ElectricalComponent
+    electrical_components = 1;
 }
 
 // Request parameters for the RPC `ListSensors`.
 // Contains filtering parameters for listing sensors.
 message ListSensorRequest {
   // Return sensors that have the specified IDs only.
   repeated uint64 sensor_ids = 1;
-
-  // Return sensors that have the specified categories only.
-  repeated frequenz.api.common.v1.microgrid.sensors.SensorCategory
-    categories = 2;
 }
 
 // Response message for the RPC `ListSensors`.
 // A message containing a list of sensors.
 message ListSensorsResponse {
-  repeated frequenz.api.common.v1.microgrid.sensors.Sensor sensors = 1;
+  repeated frequenz.api.common.v1alpha7.microgrid.sensors.Sensor sensors = 1;
 }
 
-// Request parameters for the RPC `ListConnections`.
+// Request parameters for the RPC `ListElectricalComponentConnections`.
 // Contains filtering parameters for listing connections.
-message ListConnectionsRequest {
-  // Only return connections that start from the specified component ID(s):
-  // if empty, connections with any `start` will be returned
+message ListElectricalComponentConnectionsRequest {
+  // Only return connections that start from the specified electrical
+  // component ID(s).
+  // If empty, connections with any `start` will be returned.
   repeated uint64 starts = 1;
 
-  // Only return connections that end at the specified component ID(s):
-  // if empty, connections with any `end` will be returned
+  // Only return connections that end at the specified electrical
+  // component ID(s).
+  // if empty, connections with any `end` will be returned.
   repeated uint64 ends = 2;
 }
 
-// Response message for the RPC `ListConnections`.
+// Response message for the RPC `ListElectricalComponentConnections`.
 // Contains a list of connections.
-message ListConnectionsResponse {
+message ListElectricalComponentConnectionsResponse {
   // The list of connections.
-  repeated frequenz.api.common.v1.microgrid.components.ComponentConnection
+  repeated frequenz.api.common.v1alpha7.microgrid.electrical_components.ElectricalComponentConnection
     connections = 1;
```

Comment:
Whichever route we take, I think it would be good to be consistent, so if we are using prefixes, I would make this `electrical_component_connections`, but I think I prefer the short versions, it seems like at the stage of creating/reading a message it is already obvious that we are talking about electrical components from the message and RPC names.

Reply:
OK, so rename this to `electrical_component_connections` then?

Reply:
Done.